### DISCUSSION TITLE
Feature Request: Allow using Assistants from OpenAI and Azure OpenAI at the same time

### DISCUSSION THREAD

**Discussion by strayer** [danny-avila/LibreChat#2128] [Feature Requests & Suggestions]
From what I understand, the LibreChat UI seems to be designed with the assumption that there is only ONE AI endpoint for supporting assistants. This becomes an issue when you configure Azure OpenAI for assistants, as it appears you can no longer use OpenAI itself.

Maybe having two different Assistants entries in the chat dropdown ("Azure Assistants", "OpenAI Assistants", ...) could be a solution for this.

**Comment by danny-avila** [comment]
I think it's a fair request but probably won't be a high priority for now.

> the LibreChat UI seems to be designed with the assumption that there is only ONE AI endpoint

Right, it was kind of a surprise Azure released support for assistants, I did not expect it to be an endpoint "type."

I can accommodate for this but it will be involved. I think for most use cases, either/or is satisfactory for now.
